scripts/cephscript/ceph_script.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+set +e
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+
+CEPH_DIR="/root/s3-tests"
+CEPH_INIPATH="$CEPH_DIR/seagates3conf.ini"
+CEPH_TEST_DATA="/root/cephtestfilev2.txt"
+CEPH_TEST_DATA1="/root/cephtestfilev4.txt"
+
+# Set up Virtualenv
+echo "starting virtualenv"
+source $CEPH_DIR/virtualenv/bin/activate
+
+# Export config filepath
+export S3TEST_CONF=$CEPH_INIPATH
+
+# Running ceph test with aws v2 signature
+
+export S3_USE_SIGV4=0
+
+# Log Directory
+#ceph_log_dir="/root/logs/ceph-log"
+ceph_log_dir="/root/sachit"
+mkdir -p "$ceph_log_dir"
+
+while read test_name; do
+        # reading each line
+        #echo $test_name
+
+        $CEPH_DIR/virtualenv/bin/nosetests $test_name >> $ceph_log_dir/cephrunv2.log 2>&1
+
+done < $CEPH_TEST_DATA
+
+sleep 30s
```

Comment:
should we log or print a message that script will go in sleep for 30s, to make sure we do not give impression that script is hung to the user ?